### PR TITLE
add test for Cassandra-12420 duplicate key IN clause

### PR DESCRIPTION
**Pull request by jasonstack**

### CODE REVIEW

File: paging_test.py [hunk 1]
```diff
@@ -937,6 +937,33 @@ def static_columns_with_empty_non_static_columns_paging_test(self):
         results = list(session.execute("SELECT * FROM test WHERE a IN (0, 1, 2, 3, 4)"))
         self.assertEqual([0, 1, 2, 3, 4], sorted([r.s for r in results]))
 
+    def test_select_in_clause_with_duplicate_keys(self):
+        """
+        @jira_ticket CASSANDRA-12420.
```
Comment (ptnapoleon) [inline paging_test.py]: Can you remove this trailing `.`?

**Comment by ptnapoleon** [comment]
+1 after nit

**Comment by jasonstack** [comment]
fixed. thanks
